src/server/generic_family.cc

```diff
@@ -653,7 +654,7 @@ void OpScan(const OpArgs& op_args, const ScanOpts& scan_opts, uint64_t* cursor,
 
   const auto start = absl::Now();
   // Don't allow it to monopolize cpu time.
-  const absl::Duration timeout = absl::Milliseconds(10);
+  const absl::Duration timeout = absl::Milliseconds(2);
```

Comment:
I would like to have a smaller constant here, something like 15 micro.
Lets move here to using 
base::CycleClock::Frequency()
see example in snapshot.cc
const uint64_t kCyclesPerJiffy = base::CycleClock::Frequency() >> 16;  // ~15usec.